cmd_info.go

```diff
@@ -8,13 +8,7 @@ import (
 
 // Command 'INFO' from https://redis.io/commands/info/
 func (m *Miniredis) cmdInfo(c *server.Peer, cmd string, args []string) {
```

Comment:
this one and cmdSelect don't use handleAuth, but with isValidCMD they now do. That looks like it shouldn't change. Otherwise it looks like a nice change.